source/docs/03-metadata.xml

```diff
@@ -1439,7 +1439,7 @@
                            <specDesc key="lg"/>
                         </specList>
                      </p>
-                     <p>The following example shows the encoding of the title page of Vaughan Williams' <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark the line breaks present in the original.</p>
+                     <p>The following example shows the encoding of the title page of <name ref="https://en.wikipedia.org/wiki/Ralph_Vaughan_Williams">Vaughan Williams'</name> <hi rend="italic">On Wenlock Edge</hi>. Note the use of the <gi scheme="MEI">lb</gi> element to mark new lines present in the original.</p>
```

Comment:
same here: new line beginnings ?

Reply:
Suggestion to also adjust the following `text-sample371.txt` where the `lb` elements appear at the end of every line.

Reply:
Same applies to `text-sample362.txt`

Reply:
The link is referring not referring to a picture, but to an article where no title plage is shown at the moment. Could we use a permanent url to a stable version or a link to a wiki resource here?

Reply:
It links the name of Vaughan Williams to the corresponding article about this composer.

Reply:
ah sorry, I was wrong. I just saw a link in a text about a picture and the encoding and expected (maybe also) a link to this. so, just ignore my comment!